pkg/WORKSPACE

```diff
@@ -19,6 +19,14 @@ load("//:deps.bzl", "rules_pkg_dependencies")
 
 rules_pkg_dependencies()
 
+# Find rpmbuild if it exists.
+load("//toolchains:autoconf_rpmbuild.bzl", "autoconf_rpmbuild")
+autoconf_rpmbuild(name="rules_pkg_rpmbuild")
```

Comment:
Nit:  Naming? The equivalent in `rules_python` is the [autodetecting_toolchain](https://github.com/bazelbuild/rules_python/blob/c064f7008a30f307ea7516cf52358a653011f82b/python/BUILD#L107-L125).  Perhaps a similar name should be used here?

FWIW, I didn't look very hard for other potential names.

Reply:
Yes. autoconf is a poor choice. It might be confused with autoconf from the configure tools.
I think we need rpmbuild somewhere in the name though. If everyone used the generic name 'autodetecting_toolchain' then they would clash in WORKSPACE, so people would have to load with
an alias.
I'll think about this when I review all the comments.

Reply:
I settled on rpmbuild_configure.
- putting rpmbuild before all the rpmbuild related files will be the easier to follow organization when we add something like MSI or DMG toolchains
- configure by itself is a well used term. It doesn't have the baggage of 'autoconf'